src/app/+admin/admin-access-control/epeople-registry/eperson-form/eperson-form.component.spec.ts

```diff
@@ -299,15 +294,15 @@ describe('EPersonFormComponent', () => {
 
     it ('should call the epersonFormComponent delete when clicked on the button' , () => {
       spyOn(component, 'delete').and.stub();
-      spyOn(component.epersonService, 'deleteEPerson').and.returnValue(observableOf(new RestResponse(true, 204, 'No Content')));
+      spyOn(component.epersonService, 'deleteEPerson').and.returnValue(createFailedRemoteDataObject$('No Content', 204));
```

Comment:
A very minor point... a `204` response is not a "failure".  It's a successful response, but with no content returned.  So having to use the `createFailedRemoteDataObject()` method here is a little odd.  It seems like it's necessary right now because `createSuccessfulRemoteDataObject` hardcodes the response to `200`.

This is a bit nit-picky, I know. But, it's something we may want to clean up at some point, simply because it looks odd in these specs...it makes it look like the spec is *mocking/expecting* a failure, but it's not. It's just expecting a `204` response instead of a `200`.

Maybe an alternative here is to have a `createNoContentRemoteDataObject()` for a `204`...or let the `createSuccessfulRemoteDataObject` method accept a response code param.

Reply:
You're right, that was a mistake. Fixed